sources/platform/management/subscription/node-pools.md

```diff
@@ -104,3 +108,12 @@ containing On-demand nodes, no additional steps are required.
 
 * To remove [BYON nodes](/platform/runtime/nodes/#byon-nodes) from node pools,
   delete individual nodes from their respective node pools. Refer to the documentation on [BYON nodes](/platform/runtime/nodes/#byon-nodes) for steps to delete nodes.
+
+### Using static IP to whitelist traffic from on-demand nodes
+* Knowing the IP addresses of the build machines Shippable uses can be helpful when you need them to whitelist specific IP ranges so that build nodes can freely talk to machines/tools within your VPC or network.
+
+* You can use static IP in your subscription by creating a node pool with Static IP enabled or enabling Static IP in existing node pools.
```

Comment:
This and the line above should be regular text instead of bullets. I think we should also link [creating a node pool] and [enabling static IP] to the sections above so everything is discoverable regardless of which section you end up reading first.